h2o-core/src/main/java/water/api/schemas3/GridExportV3.java

```diff
@@ -5,13 +5,16 @@
 import water.api.API;
 
 public class GridExportV3 extends SchemaV3<Iced, GridExportV3> {
-  @API(required = true, direction = API.Direction.INPUT, help = "ID of the Grid to load from the directory",
-          level = API.Level.critical)
+  @API(help = "ID of the Grid to load from the directory",
+      required = true, direction = API.Direction.INPUT, level = API.Level.critical)
   public String grid_id;
 
-  @API(required = true, direction = API.Direction.INPUT, help = "Path to the directory with saved Grid search",
-          level = API.Level.critical)
+  @API(help = "Path to the directory with saved Grid search",
+      required = true, direction = API.Direction.INPUT, level = API.Level.critical)
   public String grid_directory;
   
+  @API(help = "True if data frames should also be saved", 
```

Comment:
I think we need to be more clear what is actually saved. Users can easily confuse this for eg. CV holdout predictions or other frames that are created as a _result_ of model training.

Reply:
good call, I tried to improve the naming and docs